src/app/common/utils/sandbox.ts

```diff
@@ -0,0 +1,8 @@
+export function showFn(contractId: string, abiFn: any) {
+  return (
```

Comment:
Can you add a brief comment to document this function and in particular, make a note about stacks-increase in PoX-2.

Separately, I found this implementation a bit hard to reason through: basically we're saying show a function IF:
* it's NOT private, _and_
* it NOT stacks-increase in POX-2

IMO an implementation that uses fewer negatives might be easier to understand and maintain. Something like:

hide a function IF
* it's private OR
* it's stacks-increase in testent/mainnet pox-2

Reply:
My motivation was to have a cleaner code in the components:
`showFn(contractId, abiFn) && <FunctionView.../>`

and when using filter the abi